### DISCUSSION TITLE
Client Side Links Between Applications

### DISCUSSION THREAD

**Discussion by jaybytez** [vercel/next.js#16750] [Help] [answered]
Basic question, instead of running two applications/parts of the site in one project, we are talking about splitting the project out into two separate Next.js applications.  But if I am running two different Next.js applications, can I do Client-Side Linkage between applications?

**Comment by jamesmosier** [comment, marked as answer]
Unfortunately if they are 2 separate Next apps that won't work. Currently the `next/router` logic is dependent on all pages being a child of the root where Next gets mounted. In this case it would be 2 separate apps that are mounted at 2 different roots.
